src/shared/common/forms/accordion.jsx

```diff
@@ -25,15 +26,36 @@ function Section({ data, errors, index, items, title, updateState, page }) {
     const clickHandler = React.useCallback(event => {
         event.preventDefault();
         setVisible(!isVisible);
-    }, [isVisible]);
+        if (toggleText === 'Show') {
```

Comment:
Is there a more reliable way to determine state than relying on the text? Can we use `isVisible`?